### PR TITLE
Improve code coverage report

### PR DESCRIPTION
**Pull request by snnn**
**Description**:

A small improvement: add branch name there. So that we can filter the report by branch names. Usually we only care about the master branch.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/post-merge-jobs.yml [hunk 1]
```diff
@@ -24,22 +24,26 @@ jobs:
           #Merge the mulitple prof data into a single indexed profile data file
           llvm-profdata merge -sparse -o ort.profdata *.profraw
           #Create coverage report, output the result to 'report.json'
-          llvm-cov export -summary-only -instr-profile=ort.profdata onnxruntime_test_all -object onnxruntime_mlas_test  $(Build.SourcesDirectory)/include/onnxruntime $(Build.SourcesDirectory)/onnxruntime/core $(Build.SourcesDirectory)/onnxruntime/contrib_ops > $(Build.BinariesDirectory)/report.json
```
Comment (skottmckay) [inline tools/ci_build/github/azure-pipelines/post-merge-jobs.yml:27]: >          llvm-cov export -summary-only -instr-profile=ort.profdata onnxruntime_test_all -object onnxruntime_mlas_test   [](http://example.com/codeflow?start=0&length=120)

Are the onnxruntime_mlas_test tests included indirectly now?

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/post-merge-jobs.yml:27]: Do you mean I should remove the following line: 
https://github.com/microsoft/onnxruntime/blob/df57e866228f096ea975d92f27eb0cffbe058d0b/tools/ci_build/github/azure-pipelines/post-merge-jobs.yml#L23

Comment (skottmckay) [reply tools/ci_build/github/azure-pipelines/post-merge-jobs.yml:27]: I was just checking that onnxruntime_mlas_test was still run and included in the results. Wasn't clear to me why we replaced the calls to llvm-cov with onnxruntime_mlas_test with onnxruntime_api_tests_without_env.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/post-merge-jobs.yml:27]: Oh, that's a mistake!

**Review by skottmckay — APPROVED** [review]
:shipit:

**Pull request merged**